soundbay/utils/metadata_processing.py

```diff
@@ -101,21 +101,47 @@ def annotations_df_to_csv(annotations_dataset, dataset_name: str = 'recordings_2
     annotations_dataset.to_csv(filename, index=False)
 
 
-def merge_calls(sorted_df: pd.DataFrame) -> List[pd.Series]:
+def merge_calls(sorted_df: pd.DataFrame, overlap_pct_th: float) -> List[pd.Series]:
     """
     Args:
         sorted_df: DataFrame with sorted calls by begin_time field
```

Comment:
this function works well only for df with a single type of call. it was this way before too, but if we edit it maybe it's something that will be nice to add support to, or at least to validate before merging.
Otherwise (if the input df contains several types of calls), nothing crashes, but the results will be a mess

Reply:
Good point! 
I guess that for multiple call types we might need a different logic, e.g., in case of overlap between two different call types we would need to decide how to merge them (we would also need to have a call_type variable and decide on the type of the merged call). I think it would be interesting to work on that as a different PR if you have some time maybe we can brainstorm the logic for this function :) 
Regarding the validation I think it is important, however, the current function doesn’t account for a call_type column, do we want to add it?